json_serializable/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
-## 3.5.1-dev
+## 4.0.0-nullsafety.1
 
+- Generates null-safe code.
 - Improved error messages for unsupported types.
 - `package:json_serializable/type_helper.dart`
   - Made the third parameter to `UnsupportedTypeError` positional (optional).
+- **BREAKING** `bool defaultProvided` arg added to `TypeHelper.deserialize`.
```

Comment:
Also mention the removal of `nullable` options in favor of respecting the nullability of the types.

Reply:
Right!